src/commands/create/index.ts

```diff
@@ -68,6 +68,7 @@ export default class Create extends Command {
       required: true,
       env: 'CDA_TOKEN',
     }),
+    'request-batch-size': Flags.integer({ description: 'Limit for every single request', default: 1000 }),
```

Comment:
Love it! 
Is this aligning with the flag name we use in other CLIs?

Reply:
Yes it's the same name as contentful-import, I found out that we used to have a variable doing something similar in the context, so I got rid of it and change it for `requestBatchSize`